.github/workflows/reusable-build-on-manylinux.yml

```diff
@@ -14,6 +14,9 @@ on:
 
 jobs:
   build_on_manylinux:
+    permissions:
+      contents: write
+      packages: write
```

Comment:
We don't use GitHub packages. Is this permission required?

Reply:
Right we don't removing it.